config/config.js

```diff
@@ -25,10 +25,10 @@ module.exports = {
     dialect: "postgres",
   },
   production: {
-    username: DATABASE_URL,
+    username: DB_USER_NAME,
```

Comment:
Good work @un-anelka, next time though, we could refactor non changing config vars into a base config and only alter the changing ones for each environment